samples/colorpicker/src/com/amazon/ask/colorpicker/handlers/MyColorIsIntentHandler.java

```diff
@@ -43,16 +42,14 @@ public boolean canHandle(HandlerInput input) {
         Intent intent = intentRequest.getIntent();
         Map<String, Slot> slots = intent.getSlots();
 
-        // Get the color slot from the list of slots.
-        Slot favoriteColorSlot = slots.get(COLOR_SLOT);
+        // Get the color slot value from the user input.
+        String favoriteColor = slots.get(COLOR_SLOT).getValue();
 
         String speechText, repromptText;
-        boolean isAskResponse = false;
 
         // Check for favorite color and create output to user.
-        if (favoriteColorSlot != null) {
+        if (favoriteColor != null) {
```

Comment:
Does this fix the customer issue? If I provide an unknown color will the correct code path still be executed?

Reply:
Fix in PR, https://github.com/alexa/alexa-skills-kit-sdk-for-java/pull/160.